sdk/dapp-kit/src/components/WalletProvider.tsx

```diff
@@ -47,6 +48,9 @@ export type WalletProviderProps = {
     /** The theme to use for styling UI components. Defaults to using the light theme. */
     theme?: Theme | null;
 
+    /** Enable IOTA Names resolution */
+    iotaNames?: boolean;
```

Comment:
use descriptive names for props
```suggestion
    isIotaNamesEnabled?: boolean;
```